### PR TITLE
Credit info amendments

### PR DESCRIPTION
**Pull request by EvanBarbour3**
## Features
- Removed `Total Repayment`
- Total cost no longer includes `amount service`
- Added `Promotional` object
- Removed `Promotional` fields from credit info call, replaced with the object
- Added `displayed as` field for credit information to relate to what the fields are called in the front end

## Bug Fixes
- Fixed `/v3/merchant_api.md` as there was an invalid character

### CODE REVIEW

File: _includes/v3/merchant_api.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Field | Type | Notes
 `code` | int | For error results the code field will be always a negative integer. Codes for errors will be as described below.
 `response` | object | Response object will contain a message field describing the error.
 
-##### Standard API Errors
+##### Standard API Errors
```
Comment (srleach) [inline _includes/v3/merchant_api.md]: @on2 how did you find this? very well hidden indeed!

File: _includes/v4/objects.md [hunk 2]
```diff
@@ -123,3 +123,38 @@ Name | Required | Type | Description
...
+`$.date_end_nice` | Yes | string | The date when the promotional period ends in plain English. | Promotional End Date
+`$.deposit_amount` | Yes | int | The amount of deposit in pence used for the credit information. | Deposit
+`$.loan_amount` | Yes | int | The loan amount in pence, being the `$.order_amount` − `$.deposit_amount`. | Loan Amount
+`$.order_amount` | Yes | int | The order amount in pence. | Price
+`$.term` | Yes | int | The number of months before the promotional period ends. | N/A
+`$.total_cost` | Yes | int | The total cost in pence, being `$.order_amount` + `$.customer_settlement_fee`. | Total Repayable
+
+The promotional object is used when there is a promotional offer for
+a particular product, e.g. Buy Now Pay Later, as you can settle
+early and pay less fees.
```
Comment (srleach) [inline _includes/v4/objects.md]: It's not actually reducing the fees - it would however be reducing the interest. ` and pay less interest.`

**Comment by srleach** [comment]
@EvanBarbour3 - I'm happy with the changes subject to the change being made to the `pay less fees` part commented above.

**Comment by srleach** [comment]
Also, does this have an associated ticket# ?

**Comment by EvanBarbour3** [comment]
Done is done. Does not have a ticket, but was partially part of "product landing page refactor".

**Comment by srleach** [comment]
Dankë.
